api-docs/swagger-v1.2.yml

```diff
@@ -130,6 +130,7 @@ paths:
         - $ref: '#/parameters/package_size_min'
         - $ref: '#/parameters/package_size_max'
         - $ref: '#/parameters/package_size_unit'
+        - $ref: '#/parameters/maximum_age'
```

Comment:
Changed it, it doesn't sound correct in my head though, maybe something to consider for API 2.0